livre/core/render/FrameInfo.h

```diff
@@ -31,14 +31,34 @@ namespace livre
 struct FrameInfo
 {
     LIVRECORE_API FrameInfo( const Frustum& frustum,
-                             const uint32_t frameId );
+                             const uint32_t frameId,
+                             const uint32_t id );
 
     Frustum frustum; //!< The current frustum.
-    uint32_t frameId ; //!< The current frame id.
+    uint32_t frameId ; //!< The current frame id for livre data sources.
+    uint32_t id; //!< The unique id for each rendered frame
```

Comment:
Damn github history ! This is me being explaining the 3rd time. One is the Livre frameId ( 0 - 262k ). The other is just the counter for the rendered frame in Eq.